### DISCUSSION TITLE
Make a portal only visible internally

### DISCUSSION THREAD

**Discussion by panaras10** [chatwoot/chatwoot#8134] [Chatwoot APIs]
### Is your feature or enhancement related to a problem? Please describe.

Is there a way to create a portal that is visible only internally to logged in chat users? Use it like an internal wiki?

### Describe the solution you'd like

Is there a way to create a portal that is visible only internally to logged in chat users? Use it like an internal wiki?

**Comment by pranavrajs** [comment]
Thanks for the suggestion. I'm moving this to the discussion as we will not be prioritizing this immediately. However we would be shipping protected pages soon.
